src/Compilers/CSharp/Test/Emit/CodeGen/UnsafeTests.cs

```diff
@@ -9539,6 +9539,92 @@ .maxstack  2
 ");
         }
 
+        [Fact, WorkItem(49639, "https://github.com/dotnet/roslyn/issues/49639")]
+        public void CompareToNullWithNestedUnconstrainedTypeParameter()
```

Comment:
>CompareToNullWithNestedUnconstrainedTypeParameter [](start = 20, length = 49)

I assume a non-generic test is covered elsewhere.

Reply:
Yes: [https://github.com/dotnet/roslyn/blob/master/src/Compilers/CSharp/Test/Emit/CodeGen/PatternTests.cs#L4463](https://github.com/dotnet/roslyn/blob/master/src/Compilers/CSharp/Test/Emit/CodeGen/PatternTests.cs#L4463)

---
In reply to: [541420114](https://github.com/dotnet/roslyn/pull/49915#discussion_r541420114) [](ancestors = 541420114)